frontend/apps/marketing/src/app/globals.css

```diff
@@ -0,0 +1,42 @@
+:root {
```

Comment:
Please note, this file will be modified in a followup PR away from the starter version.